tests/approaches/test_nsrt_learning_approach.py

```diff
@@ -117,6 +122,14 @@ def test_nsrt_learning_approach():
                               additional_settings={
                                   "compute_sidelining_objective_value": True,
                               })
+    approach = _test_approach(env_name="blocks",
+                              approach_name="nsrt_learning",
+                              try_solving=False,
+                              no_loading=True,
+                              additional_settings={
+                                  "compute_sidelining_objective_value": True,
```

Comment:
remove `compute_sidelining_objective_value` True? it'll be slower as True, right? and it's covered by the test above